src/BenchmarksApps/Samples/Template/Controllers/StringComparerController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Template.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class StringComparerController : ControllerBase
```

Comment:
I think we should use a piece of middleware instead, of endpoint routing, to remove the mvc stack that could add a lot of noise to the profile. Endpoint routing might be the easiest way, so the dev would just edit the Startup file to edit the code to run.

Reply:
It would look like this:

```csharp
routeBuilder.Map("InvariantCultureIgnoreCase/{count}", context =>
{
    var count = context.GetRouteValue("count");
    return context.Response.WriteAsync($"Hi! {count}");
});
```